js/front/dataset/index.js

```diff
@@ -216,30 +219,18 @@ new Vue({
         checkResource(resource) {
             const url = parseUrl(resource.contentUrl);
             const resource_el = document.querySelector(`#resource-${resource['@id']}`);
-            const el = resource_el.querySelector('.format-label');
+            const el = resource_el.querySelector('.healthcheck-container');
             const checkurl = resource_el.dataset.checkurl;
-            if (url.protocol.startsWith('ftp')) {
-                el.classList.add('format-label-warning');
-                el.setTooltip(this._('The server may be hard to reach (FTP).'), true);
-            } else {
-                this.getResourceCheckStatus(resource, checkurl)
+            this.getResourceCheckStatus(resource, checkurl)
```

Comment:
Given you introduced a Availability component, it might me clean and easier to encapsulate the entire check logic in it

Reply:
Good idea but there's the slight difficulty of triggering the check only for visible cards, I'm not sure I can handle that in a component?